ADavidson02/funny-dev
# Feature/user

## Is this a fix or a feature?
Add a header component

## What is the change?
The header is now done in material ui

## What does it fix?
styling of the header

## Where should the reviewer start?
Header.jsx line 1

## How should this be tested?
run the test suite and check all tests pass

##Screenshots(if appropriate)

@ADavidson02
